### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by MattDMo** [guillermooo/sublime-undocs#254]
I've been on a programming/development hiatus for a year plus and am not up to speed on a bunch of things, so please forgive me if this has already been discussed, but what is the status of the unofficial docs at this point? I was just browsing through the [official documentation](https://www.sublimetext.com/docs/3/) and was pretty impressed with the couple of pages I glanced at, so is this project irrelevant at this point? It doesn't look like there's been a whole lot of action recently, and there are still some pretty glaring issues and omissions: 

- popups, minihtml, and CSS styling not even mentioned — [`view.show_popup()`](https://www.sublimetext.com/docs/3/api_reference.html#sublime.View) was added in build 3070 (released 17 Feb ***2015***)
- new `.sublime-color-scheme` format
- *lots* of API changes
- build system changes and error highlighting
- new settings options
- theming enhancements

and probably a bunch more. I don't have the time ATM to check if all the API functions in the [*"Missing in the official docs"*](http://docs.sublimetext.info/en/latest/reference/api.html#missing-in-the-official-docs) section have since been added (at least `sublime.Window.set_layout()` has not), so that section may still be useful, but at least according to [this issue](https://github.com/SublimeTextIssues/Core/issues/2144) the "undocumented API" may be that way for a reason and subject to change or deletion at any time - @FichteFoll could you add any clarity to that?

So, does this project still have a *raison d'être*, or should it be clearly marked in the header of every page as containing potentially obsolete information and direct visitors to the official docs?

**Comment by FichteFoll** [comment]
There is still *a lot* do be done with the project. Our roadmap is huge, which is also problematic because noone of us finds the time to work on it.

Currently, the first thing that needs attention imo is hosting because rtd isn't too quick on rebuilding and atm I'd rather have CI and auto-deploy through travis and hosting on Github pages. Once that is done, the next step on our agenda is restructuring so that introduction (kind of like a "user guide" style) and reference (with all the details) are separate. And then start working on that.

The official docs have certainly improved over the past months and made some of our pages obsolete, but I haven't compared them for details that are omitted in the official docs. Besides, the aforementioned structure would be largely different from the current official docs, which we would however use to defer users to in some cases, e.g. for build systems or the API docs.

To summarize: The project definitely still has a raison d'être, but is also time consuming - time that we just don't have at the moment. (I can't even stay up to date with Package Control Channel reviews.)

BTW, we're also looking to ways to allow for donations so that we can justify setting aside a lot of our time for the project. Of course, documentation is important, but it is also not exactly the most fun part about software engineering.
